examples/seismic/viscoacoustic/operators.py

```diff
@@ -0,0 +1,219 @@
+import sympy as sp
+import numpy as np
+
+from devito import (Eq, Operator, VectorTimeFunction, TimeFunction, NODE,
+                    div, grad)
+from examples.seismic import PointSource, Receiver
+
+
+def blanch_symes(model, geometry, v, p, **kwargs):
+    """
+    Stencil created from from Blanch and Symes (1995) / Dutta and Schuster (2014)
+    viscoacoustic wave equation.
+
+    https://library.seg.org/doi/pdf/10.1190/1.1822695
+    https://library.seg.org/doi/pdf/10.1190/geo2013-0414.1
+
+    Parameters
+    ----------
+    model : Model
+        Object containing the physical parameters.
+    geometry : AcquisitionGeometry
+        Geometry object that contains the source (SparseTimeFunction) and
+        receivers (SparseTimeFunction) and their position.
+    v : VectorTimeFunction
+        Particle velocity.
+    p : TimeFunction
+        Pressure field.
+    """
+    space_order = kwargs.get('space_order')
+    save = kwargs.get('save')
+
+    s = model.grid.stepping_dim.spacing
+    irho = model.irho
+    vp = model.vp
+    damp = model.damp
+    qp = model.qp
+    f0 = geometry._f0
+
+    # The stress relaxation parameter
+    t_s = (sp.sqrt(1.+1./qp**2)-1./qp)/f0
+
+    # The strain relaxation parameter
+    t_ep = 1./(f0**2*t_s)
+
+    # The relaxation time
+    tt = (t_ep/t_s)-1.
+
+    # Density
+    rho = 1. / irho
+
+    # Bulk modulus
+    bm = rho * (vp * vp)
+
+    # Memory variable.
+    r = TimeFunction(name="r", grid=model.grid, staggered=NODE,
+                     save=geometry.nt if save else None,
+                     time_order=1, space_order=space_order)
+
+    # Define PDE
+    pde_v = v - s * irho * grad(p)
+
+    u_v = Eq(v.forward, damp * pde_v)
+
+    pde_r = r - s * (1. / t_s) * r - s * (1. / t_s) * tt * bm * div(v.forward)
+
+    u_r = Eq(r.forward, damp * pde_r)
+
+    pde_p = p - s * bm * (tt + 1.) * div(v.forward) - s * r.forward
+
+    u_p = Eq(p.forward, damp * pde_p)
```

Comment:
That's a lot of empty lines, maybe remove the empty line between `pde_` and `u_`